docs/source/interfaces/PyChop.rst

```diff
@@ -163,6 +163,38 @@ and :math:`E_f` are the incident and scattered neutron energies.
 
 The flux is obtained from lookup tables of measured (white-beam) flux on each instrument.
 
+
+Sample contribution
+-------------------
+
+Although the contribution of a sample to the resolution of direct-geometry chopper spectrometers
+is usually negligible,
+it is currently included by calculating of the variance of the time-of-flight due to the size
+of the sample.
+So far only two shapes are supported: plate and thin annulus.
+The broadening caused by a plate sample is calculated as proportional
+to :math:`\frac{1}{12} w^2`, where :math:`w` is the width of the plate.
+The factor :math:`\frac{1}{12}` comes from the variance of a uniform distribution
+
+.. math:: \int^{\frac{1}{2}}_{-\frac{1}{2}} x^2 dx = \frac{1}{12}
+
+.. figure:: sample-annulus-variation.png
+   :alt: /images/Pychop-annulus-shape.png
```

Comment:
I think the above two fields should be swapped around so it should be:

```rst
.. figure:: /images/PyChop-annlus-shape.png
    :alt: sample-annulus-variation.png
```

as the doc-test is complaining that it can't read the file `sample-annulus-variation.png` (which doesn't exist)...

Reply:
Good catch! Thanks